server/src/main/java/io/deephaven/server/arrow/ArrowFlightUtil.java

```diff
@@ -507,7 +518,15 @@ public void handleMessage(@NotNull final BarrageProtoUtil.MessageInfo message) {
                                 .onError(listener)
                                 .submit(() -> {
                                     metrics.queueNanos = System.nanoTime() - queueStartTm;
-                                    final BaseTable<?> table = tableExport.get();
+                                    Object export = tableExport.get();
```

Comment:
I think you should change the type of the `ExportObject` to be `<?>`, since we're being more careful about validation downstream.